pkgpanda/build/__init__.py

```diff
@@ -37,8 +39,14 @@ def __init__(self):
         self.environment = dict()
         self.container = str()
 
-    def run(self, cmd):
-        docker = ["docker", "run"]
+    def run(self, name, cmd):
+        docker = ["docker", "run", "--name={}".format(
```

Comment:
`"--name={}-{}".format(name, ''.join(random.choice(string.ascii_lowercase) for _ in range(10)` as another way to write this?